docs/src/adoc/index.adoc

```diff
@@ -553,22 +553,44 @@ that returns a result set from the database.
 include::{exampledir}/StatementsTest.java[tags=query]
 ----
 
-For single rows, you can use `findOnly()`, which expects exactly one row (or
-throws an exception):
+To get a single row from a query, there are a few possible methods to use,
+depending on the number of rows that might be in the result set.
+
+Call `one()` when the result contains exactly one row. Throws an exception if
```

Comment:
Caller doesn't know the result cardinality, but they can have their expectations about that:

```suggestion
Call `one()` when you expect the result to contain exactly one row. Throws an exception if
```

(same below in other cases)

Reply:
Good idea. I incorporated this rewording as well as some clarifications about return types in the latest commit.